crates/ludtwig/src/check/rules/twig_prefer_sw_extends_over_extends.rs

```diff
@@ -0,0 +1,65 @@
+use ludtwig_parser::syntax::typed::{AstNode, TwigExtends};
+use ludtwig_parser::syntax::untyped::SyntaxNode;
+
+use crate::check::rule::{CheckResult, Rule, RuleExt, RuleRunContext, Severity};
+
+pub struct RuleTwigShopwareExtends;
+
+impl Rule for RuleTwigShopwareExtends {
+    fn name(&self) -> &'static str {
+        "twig-prefer-sw-extends-over-extends"
+    }
+
+    fn check_node(&self, node: SyntaxNode, _ctx: &RuleRunContext) -> Option<Vec<CheckResult>> {
+        let twig_extends = TwigExtends::cast(node)?;
+
+        // not a valid twig operator
+        let result = self
+            .create_result(Severity::Error, "prefer 'sw_extends' over 'extends'")
+            .primary_note(
+                twig_extends.syntax().text_range(),
+                "help: change 'extends' to 'sw_extends'",
+            )
+            .suggestion(
+                twig_extends.syntax().text_range(),
+                "sw_extends",
```

Comment:
This is kind of problematic, because you are replacing the whole TwigExtends Syntax Node here (`{% extends ... %}` and not only the word `extends`. But I totally understand that it isn't easy to get only the extends word without some more AST API helper methods 😉